src/%IPM/ResourceProcessor/Abstract.cls

```diff
@@ -225,35 +225,40 @@ Method OnGetUniqueName(Output pUniqueName)
 {
 }
 
-Method GetSourceControlInfo(Output pInfo As %IPM.PackageManager.Developer.Extension.SourceControl.ResourceInfo) As %Status
+Method GetSourceControlInfo(Output pInfo As %IPM.ExtensionBase.SourceControl.ResourceInfo) As %Status
 {
-	Set pInfo = ##class(%IPM.PackageManager.Developer.Extension.SourceControl.ResourceInfo).%New()
+	Set pInfo = ##class(%IPM.ExtensionBase.SourceControl.ResourceInfo).%New()
 	Quit $$$OK
 }
 
-/// Evaluates an expression in an attribute
+
+/// Evaluates an expression in a provided string. <br />
+/// These special expressions are case-insensitive. <br />
 /// Current valid expressions:
-/// {$namespace} - the current namespace
-/// {$mgrdir} - the instance's mgr directory
-/// {$cspdir} - the instance's root CSP directory
-/// {$root} - the resource's module's root directory
-/// {name} - the name of the resource
-/// These special expressions are case-insensitive.
-Method %Evaluate(pAttrValue) As %String [ Internal ]
+/// <ul>
+/// 	<li>{$root} - the resource's module's root directory </li>
+/// 	<li>{name} - the name of the resource </li>
+/// </ul>
+/// In addition to these, look at expressions supported by %EvaluateSystemExpression
+/// in class <class>%IPM.Utils.Module</class>.
+Method %Evaluate(pAttrValue As %String) As %String [ Internal ]
 {
-	Set tAttrValue = pAttrValue
-	If $Data(..Params) {
-		Merge tParams = ..Params
-		Do ##class(%IPM.Storage.ModuleSetting.Default).GetCustomParameters(.customParams,..ResourceReference.Module, .tParams)
```

Comment:
This seems like a merge issue. Lost this logic at the beginning of the method before %Evaluate. As it was, custom parameters should get evaluated before the defaults below of the OOTB parameters/expressions